action/execution.go

```diff
@@ -109,6 +109,11 @@ func (ex *Execution) Payload() []byte { return ex.data }
 // AccessList returns the access list
 func (ex *Execution) AccessList() types.AccessList { return ex.accessList }
 
+// SetAccessList set the access list
+func (ex *Execution) SetAccessList(list types.AccessList) {
+	ex.accessList = list
```

Comment:
latest master added `NewExecutionWithAccessList`, pls rebase and use it

Reply:
ok, done